wrapper/src/test/java/integration/container/tests/AutoscalingTests.java

```diff
@@ -127,8 +128,11 @@ public void test_pooledConnectionAutoScaling_setReadOnlyOnOldConnection()
       }
 
       final Connection newInstanceConn;
+      final String instanceClass =
+          auroraUtil.getDbInstanceClass(TestEnvironment.getCurrent().getInfo().getRequest());
       final TestInstanceInfo newInstance =
-          auroraUtil.createInstance("auto-scaling-instance");
+          auroraUtil.createInstance(instanceClass, "auto-scaling-instance");
```

Comment:
Before these changes, the new instance was added to the TestEnvironment in a convoluted way: an earlier call to AuroraTestUtility would pass in the TestEnvironment instance list, which the class would then assign to an internal instance variable. Then, in this call, the instance list would be updated. In this PR we removed the internal instance variable. Should I change the method signature to require the TestEnvironment instance list to be passed in so that we can update it inside the function instead of relying on callers to update it themselves? It would be convenient but it also would be a side effect of calling the function